### PR TITLE
Add another output format: table

### PR DESCRIPTION
**Pull request by jirutka**
Example output:

```
/*!
 * Bundled Libraries
 *
 * | Name               | Version | License(s)   | Publisher           | Source                                           |
 * | ------------------ | ------- | ------------ | ------------------- | ------------------------------------------------ |
 * | anser              | 1.4.9   | MIT          | Ionică Bizău        | https://github.com/IonicaBizau/anser             |
 * | buffer-from        | 1.1.1   | MIT          |                     | https://github.com/LinusU/buffer-from            |
 * | highlightjs        | 9.12.0  | BSD-3-Clause |                     |                                                  |
 * | is-plain-obj       | 1.1.0   | MIT          | Sindre Sorhus       | https://github.com/sindresorhus/is-plain-obj     |
 * | katex              | 0.11.0  | MIT          |                     | https://github.com/KaTeX/KaTeX                   |
 * | marked             | 0.7.0   | MIT          | Christopher Jeffrey | https://github.com/markedjs/marked               |
 * | minimist           | 1.2.0   | MIT          | James Halliday      | https://github.com/substack/minimist             |
 * | minimist-options   | 4.0.2   | MIT          | Vadim Demedes       | https://github.com/vadimdemedes/minimist-options |
 * | minimist-options   | 4.0.2   | MIT          | Sindre Sorhus       | https://github.com/sindresorhus/arrify           |
 * | nodom              | 2.4.0   | ISC          | Juha Lindstedt      | https://github.com/pakastin/nodom                |
 * | source-map         | 0.6.1   | BSD-3-Clause | Nick Fitzgerald     | https://github.com/mozilla/source-map            |
 * | source-map-support | 0.5.13  | MIT          |                     | https://github.com/evanw/node-source-map-support |
 */
```

### CODE REVIEW

**Comment by jirutka** [comment]
I forgot about sorting, I’ll push another commit in a moment…

File: README.md [hunk 1]
```diff
@@ -15,7 +15,11 @@ let license = require('rollup-plugin-node-license');
 rollup({
     ...
     plugins: [
-        license()
+        license({
+            // Format of the injected license comment(s).
+            // Options: "jsdocs" (default), or "table".
+            format: 'jsdocs'
```
Comment (PepsRyuu) [inline README.md]: I'd change this from "jsdocs" to "jsdoc", to make it consistent with JSDoc.

File: index.js [hunk 2]
```diff
@@ -13,7 +14,43 @@ function checkLicense (path) {
...
+    let rows = data.map(({ pkg, lic }) => {
+        let source = ellipsis(lic.repository || lic.email, 65);
+        return [pkg.name, pkg.version, lic.licenses, ellipsis(lic.publisher, 30), source];
+    });
+    rows.sort(([a], [b]) => a.localeCompare(b));
+    rows.unshift(['Name', 'Version', 'License(s)', 'Publisher', 'Source']);
+
+    return [
+        '/*!',
+        ' * Bundled Libraries',
```
Comment (PepsRyuu) [inline index.js]: Not sure I like the phrase "Bundled Libraries". Perhaps something like "NPM Package Licenses"?

**Comment by PepsRyuu** [comment]
Works really well! Awesome contribution! 😊

File: index.js (hunk 2 above)
Comment (jirutka) [reply index.js]: I think that “Bundled Libraries” communicate better that the names libraries are _bundled_ (included) in the source. “NPM Package Licenses” is unclear to me; licenses of what packages? Those bundled in the file, used to build this file, or external packages that this file depends on?

File: README.md (hunk 1 above)
Comment (jirutka) [reply README.md]: Ok, updated.

File: index.js (hunk 2 above)
Comment (PepsRyuu) [reply index.js]: Hmm good point. "Bundled NPM Dependencies/Packages/Modules"?

Just the word "Libraries" I'm mostly reluctant to use, as it's not consistent with NPM terminology. :) I'm aware it's a small thing, but want to make sure what's happening is properly communicated to whoever reads the bundle.

Comment (jirutka) [reply index.js]: Hmm, maybe make it configurable? :smile: But we have to decide about the default anyway. It’s hard to decide but I think that I prefer “Bundled npm Packages” (“npm” is actually not an acronym, it should be written as “npm”, not “NPM”).

Comment (PepsRyuu) [reply index.js]: No need for configuration I don't think. Just having a look over the npm website, and I never noticed that they never use uppercase letters. 😂 "Bundled npm packages" will work perfectly I think, avoiding the capital P as it seems out of place now. :P

Comment (jirutka) [reply index.js]: Okay, I’ve updated it.

**Comment by PepsRyuu** [comment]
Released in ```0.2.0```. Thanks again! 😊

**Pull request merged**